backend/src/test/java/gov/cdc/usds/simplereport/service/TestResultUploadServiceIntegrationTest.java

```diff
@@ -0,0 +1,137 @@
+package gov.cdc.usds.simplereport.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import gov.cdc.usds.simplereport.db.model.Facility;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.db.model.TestResultUpload;
+import gov.cdc.usds.simplereport.db.model.auxiliary.UploadStatus;
+import gov.cdc.usds.simplereport.db.repository.TestResultUploadRepository;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.data.auditing.AuditingHandler;
+import org.springframework.data.auditing.DateTimeProvider;
+import org.springframework.data.domain.Page;
+
+class TestResultUploadServiceIntegrationTest extends BaseServiceTest<TestResultUploadService> {
+
+  public static final UUID REPORT_ID_1 = UUID.randomUUID();
+  public static final UUID REPORT_ID_2 = UUID.randomUUID();
+  public static final UUID REPORT_ID_3 = UUID.randomUUID();
+  @MockBean DateTimeProvider dateTimeProvider;
+
+  @SpyBean private AuditingHandler handler;
+
+  @Autowired private TestResultUploadRepository uploadRepository;
+
+  @Autowired private OrganizationService organizationService;
+
+  @Autowired private TestResultUploadService testResultUploadService;
+
+  @BeforeEach
+  void setup() {
+    mockCreationTime("2020-01-01 00:00");
+    initSampleData();
+    Organization currentOrganization = organizationService.getCurrentOrganization();
+    Facility facility = _dataFactory.createValidFacility(currentOrganization);
+
+    mockCreationTime("2020-02-17 00:00");
+    uploadRepository.save(
+        new TestResultUpload(
+            REPORT_ID_1, UploadStatus.SUCCESS, 30, currentOrganization, null, null));
+
+    mockCreationTime("2021-02-17 00:00");
+    uploadRepository.save(
+        new TestResultUpload(
+            REPORT_ID_2, UploadStatus.PENDING, 20, currentOrganization, null, null));
+
+    mockCreationTime("2022-02-17 00:00");
```

Comment:
should all these mock creation times be the same date/time?

Reply:
oh wow nvm I misread the years!

Reply:
@mehansen I did the exact same thing when I reviewed this PR 🤣

Reply:
haha the covid years all blend into each other